apps/dashboard/src/components/side-navigation/side-navigation.tsx

```diff
@@ -37,6 +39,7 @@ const NavigationGroup = ({ children, label }: { children: ReactNode; label?: str
 export const SideNavigation = () => {
   const { subscription, daysLeft, isLoading: isLoadingSubscription } = useFetchSubscription();
   const isFreeTrialActive = subscription?.trial.isActive;
```

Comment:
Nit-picky one: `isTrialActive`. It took me 5 seconds to process what is a free trial, since a trial is always free 😄